### ISSUE TITLE
Back Transform Logit Values

### ISSUE THREAD

**Issue by markdane** [MEP-LINCS/MEPDataExplorer#10]
The features that are logits should be backtransformed to proportions. If this is not going to happen, they need to be described as logits in the CuratedFeatures file.

**Comment by kdaily** [comment]
Isn't the inverse of the logit function the logistic function? If so, what base is used? I think in either case, the values need to be explained (e.g. are they probabilities or just log ratios)? I just find it a bit messy to determine which columns to invert based on their name, but can do it if needed.

**Comment by markdane** [comment]
Yes. The base used is 2.

The values are proportions of the populations

We need to show them as proportions. I can add the back transformed versions of the biologically relevant features in the next release but that won't happen till after the F2F meeting. You can skip this for now if you think that's best.

**Comment by kdaily** [comment]
Check out the branch in PR #12 to see if this works properly, @markdane.
